Modules/posixmodule.c

```diff
@@ -17878,6 +18336,30 @@ all_ins(PyObject *m)
 #endif
 #endif  /* HAVE_EVENTFD && EFD_CLOEXEC */
 
+#ifdef HAVE_STATX
+    if (PyModule_AddIntMacro(m, STATX_TYPE)) return -1;
```

Comment:
nit: Any reason not to alphabetize this list? 

I like having either a comment why not alphabetical or to alphabetize for easier later code reading "is this member in the list" when trying to add more